### ISSUE TITLE
Tips Bugs + Streaks

### ISSUE THREAD

**Issue by SuruPa00** [BIDMCDigitalPsychiatry/LAMP-platform#389]
We've ran into a few tips related bugs that LA has reported when trying to get their mindLAMP set up.

LA noted that streaks arent appearing after DBT diary card is completed - can you check on this?

When you add a tip w multiple tips inside and add files associated with each individual inside tip, the loading wheel never ends when you hit save - this also occurs whenever you embed anything in them using markdown, even with a single tip

Tips links also arent able to be clicked on - you just keep clicking on them endlessly - both on app and on phone app - only 3D touch on iphone works to open a link

**Comment by ZCOEngineer** [comment]
@SuruPa00 We shall check this and reply

**Comment by sarithapillai8** [comment]
LA noted that streaks arent appearing after DBT diary card is completed - can you check on this?

@avaidyam Could you please confirm steaks are required for DBT and other activities. We implemented that for survey only.

We can check on other issues.

**Comment by avaidyam** [comment]
@sarithapillai8 Yes - it should be very little effort to enable Streaks for all activities now (using Activity ID, not the Spec) - especially for DBT Diary Card.

**Comment by sarithapillai8** [comment]
@SuruPa00 
This is updated in production. Please check.
